GDSCommon-Demo/GDSCommon-DemoTests/GDSInformationViewControllerTests.swift

```diff
@@ -1,6 +1,7 @@
 import GDSCommon
 import XCTest
 
+@MainActor
```

Comment:
Please can we add `@MainActor` to the methods rather than the class itself? The superclass `XCTestCase` isn't a main actor, so adding it here might produce unexpected behaviour.

Interestingly enough, I was chatting to [Matt Massicotte](https://www.massicotte.org/mistakes-with-concurrency) about this exact problem yesterday, apparently there's been discussion about adding a compiler exception for it 👀